### DISCUSSION TITLE
Open my site

### DISCUSSION THREAD

**Discussion by Mr-Coquin** [NativePHP/laravel#382] [Q&A] [answered]
Good day. Could you please let me know if it's possible to build an application that would open an already developed and hosted Laravel website? I couldn't quite understand this from the documentation. If it's possible, how can I do that?

**Comment by simonhamp** [comment]
Do you intend to simply run the built application within the installable browser environment?

**Reply by Mr-Coquin** [reply]
How I do it in the "electron" program: In the file "main.js" I write win.loadURL('https://my_site'); . Can you do it like this?

**Reply by simonhamp** [reply]
I just want to reality check... the goal of NativePHP is to load the built-in PHP application, not an external site... if that's what you need, why not just use Electron directly?

**Reply by Mr-Coquin** [reply]
> I just want to reality check... the goal of NativePHP is to load the built-in PHP application, not an external site... if that's what you need, why not just use Electron directly?

External site. I wonder if there is such a possibility.

**Reply by simonhamp** [reply, marked as answer]
Yes, it would be possible. It's not possible right now, but I can't imagine it would be complicated to add.

I can see it being beneficial if _parts_ of your overall app experience you want to offload to a server-based application in a mixed setup where some of your apps functionality is local-/offline-first and other parts rely on access to the web.

But if you're not using the NativePHP parts at all, then there's no point using NativePHP for this... it would just add unnecessary risk/bloat.

**Reply by Mr-Coquin** [reply]
> Yes, it would be possible. It's not possible right now, but I can't imagine it would be complicated to add.
> 
Thank you. I understand. Sad :*(

**Comment by fuzqing** [comment]
You can use Wails to do this, and it's really straightforward. Just update the content in `frontend/index.html` to `<meta http-equiv="refresh" content="0;url=https://example.com/" />`.
If you need more features, you can check out [Pake](https://github.com/tw93/Pake).
